_includes/footer-plain.html

```diff
@@ -29,9 +29,7 @@
             <li><a class="white-text " href="/careers/internships">Internships</a>
               <ul>
               <li><a class="white-text " href="/gsoc/2020">GSoC 2020</a></li>
-              <li><a class="white-text " href="/gsod/2020">GSoD 2020</a></li>
               <li><a class="white-text " href="/communitybridge/2020">CommunityBridge 2020</a></li>
-              <li><a class="white-text " href="/mlhfellowship/2020">MLH Fellowship 2020</a></li>
```

Comment:
_I think_ this is still an open possibility. I'm not sure as I think their POD program is new (?), so they might be launching more than one program (?). It'll be good to get confirmation of our rejection before removing this one.

Reply:
Lets keep on hold until we hear from MLH Team.